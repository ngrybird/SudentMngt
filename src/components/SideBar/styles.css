.sidebar {
  width: 100px;

  background: #383b54;
  float: left;
}
.logo {
  height: 80px;
  padding-top: 20px;
  padding-left: 25px;
  background: #7053ed;
}
