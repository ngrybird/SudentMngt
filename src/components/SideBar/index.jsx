import React from "react";
import logo from "../../images/logo.png";
import "./styles.css";

class SideBar extends React.Component {
  render() {
    return (
      <div className="sidebar">
        <div className="logo">
          <img src={logo} height="54px" width="54px" alt="logo" />
        </div>
      </div>
    );
  }
}
export default SideBar;
