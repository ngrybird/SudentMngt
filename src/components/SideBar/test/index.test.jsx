import React from "react";
import { render } from "@testing-library/react";
import SideBar from "../index";

describe("<SideBar/>", () => {
  it("Verify sidebar rendered", () => {
    const { container } = render(<SideBar />);

    expect(container).toMatchSnapshot();
  });
});
