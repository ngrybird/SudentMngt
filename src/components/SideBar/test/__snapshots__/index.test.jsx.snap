// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideBar/> Verify sidebar rendered 1`] = `
<div>
  <div
    class="sidebar"
  >
    <div
      class="logo"
    >
      <img
        alt="logo"
        height="54px"
        src="logo.png"
        width="54px"
      />
    </div>
  </div>
</div>
`;
